easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb

```diff
@@ -0,0 +1,22 @@
+name = 'pgi'
+version = '15.7'
+year = '2015'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['pgilinux-'+year+'-%(version_major)s%(version_minor)s-x86_64.tar.gz']
+
+gnu = 'GNU'
+gnuver = '4.9.2-2.25'
+versionsuffix = '-%s-%s' % (gnu, gnuver)
+
+dependencies = [(gnu, gnuver)]
+
+# license file
+import os
```

Comment:
it's time we introduce an additional constant `HOME`, so we can get rid of these `import` statements...